### DISCUSSION TITLE
Wallet Opening - No Way to get back to main page

### DISCUSSION THREAD

**Discussion by tcrminingrocks** [Chia-Network/chia-blockchain#2197] [General]
HI! I am a newbie, but there is no way to close the page when you open the wallet key.  Is there a work around?  No close button, no X button in the corner, when I hit close it wants to shut down, and if i exit it just comes back to the wallet id page.  Any work around or whatever?  I hit reload, and then it just hangs.  

Do not know what to do next. Yes, I am new to do.

**Comment by tcrminingrocks** [comment]
![image](https://user-images.githubusercontent.com/78711405/115411533-8c53e300-a1c1-11eb-87ea-1b14a66de463.jpeg)![image](https://user-images.githubusercontent.com/78711405/115411534-8c53e300-a1c1-11eb-93e7-0aba900dcfd9.jpeg)

**Comment by tianxinya2021** [comment]
Yes, I also felt frustrated when I go to 'key' page, but find no way to get out of the key page. 
I had to quit, causing issues in plotting processing.
Suggestion: please add a option bar on the left side as in other pages to allow people to make choices after viewing the key page. Thanks.

**Comment by tcrminingrocks** [comment]
I reboot and I’m stuck at the key page

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
